app/src/renderer/components/governance/PageGovernance.vue

```diff
@@ -1,22 +1,19 @@
 <template>
-  <tm-page class="governance" data-title="Governance"
-    ><template slot="menu-body">
-      <tm-balance :tabs="tabs">
-        <tm-btn
-          id="propose-btn"
-          :disabled="!connected"
-          :value="connected ? 'Create Proposal' : 'Connecting...'"
-          color="primary"
-          @click.native="onPropose"
-        />
-      </tm-balance>
-      <tool-bar
-        ><a v-tooltip.bottom="'Search'" @click="setSearch()"
-          ><i class="search material-icons">search</i></a
-        ></tool-bar
-      >
-    </template>
-    <modal-search type="proposals" />
+  <tm-page
+    :tabs="tabs"
+    :dataset="proposalList"
```

Comment:
```suggestion
    :dataset="proposalIds"
```

Reply:
I like it more, and it makes me think of the `dataset` property name

Reply:
Ok I've seen in every other places I was passing the whole dataset, here I was wrong in getting only the keys, so I was thinking of keeping the name but instead of keys I'll pass the Object.values, so it behaves equally along the pages

Reply:
Is `dataset` a dataset or a set of ids? can't follow ^^'